### ISSUE TITLE
Kibana v5 support?

### ISSUE THREAD

**Issue by Trouble123** [jprante/elasticsearch-jdbc#896]
Hi
Is kibana v5 support planning on being implemented? When is this expectes, when it's GA or earlier?

Thanks

**Comment by jprante** [comment]
JDBC importer never supported Kibana, but I assume you mean Elasticsearch 5, which is in alpha stage and still moves with fast pace. It will bring a plethora of changes and it will take a while after GA to get JDBC importer to support them.

**Comment by Trouble123** [comment]
Sorry, yes i meant Elasticsearch 5.

On that topic, we are looking at forking the JDBC importer to use it with SearchGuard
(https://github.com/jprante/elasticsearch-jdbc/issues/847), but will wait for v5, so we dont have to re-do it.

**Comment by Trouble123** [comment]
G'day

With the release of elasticsearch v5 today, do you have an expected time frame to support this version?

Thanks

**Comment by spiritonly** [comment]
@jprante install this plugin to elasticsearch v5 failed.  any plan to support v5? thank you!

**Comment by jprante** [comment]
Yes, I intend to support Elasticsearch 5. But there is no ETA. Many things have changed and there are also new APIs. HTTP Client is disappointing, not compatible to TransportClient API.
